go/libraries/doltcore/schema/typeinfo/typeinfo.go

```diff
@@ -181,13 +181,13 @@ func FromSqlType(sqlType sql.Type) (TypeInfo, error) {
 		case gmstypes.PolygonType{}.String():
 			return &polygonType{sqlType.(gmstypes.PolygonType)}, nil
 		case gmstypes.MultiPointType{}.String():
-			return &multipointType{}, nil
+			return &multipointType{sqlType.(gmstypes.MultiPointType)}, nil
 		case gmstypes.MultiLineStringType{}.String():
-			return &multilinestringType{}, nil
+			return &multilinestringType{sqlType.(gmstypes.MultiLineStringType)}, nil
 		case gmstypes.MultiPolygonType{}.String():
-			return &multipolygonType{}, nil
+			return &multipolygonType{sqlType.(gmstypes.MultiPolygonType)}, nil
 		case gmstypes.GeomCollType{}.String():
-			return &geomcollType{}, nil
+			return &geomcollType{sqlType.(gmstypes.GeomCollType)}, nil
 		case gmstypes.GeometryType{}.String():
 			return &geometryType{sqlGeometryType: sqlType.(gmstypes.GeometryType)}, nil
```

Comment:
what is this change for?

Reply:
This is something I should've implemented when I was originally implementing `MultiPoint`, `MultiLineString`, `MultiPolygon`, and `GeometryCollection`. When defining columns over these types, we're able to specify a default SRID, and `sqlType` is what remembers the columns user defined SRID.

Example: `create table t (p point srid 0)`

Having a SRID defined is required for Spatial Indexes to work.